src/App.js

```diff
@@ -1,22 +1,23 @@
-import React from "react"
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom"
-import Error from "./Error/Error"
-import Menu from "./Menu/Menu"
-import Home from "./Home/Home"
-import Departments from "./Departments/Departments"
-import Department from "./Department/Department"
-
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Error from './Error/Error';
+import Login from './Login/Login';
+import Departments from './Departments/Departments';
+import Department from './Department/Department';
+import Employees from './Employees/Employees';
+import Employee from './Employee/Employee';
 
 function App() {
   return (
     <Router>
       <div className="App">
-        <Menu/>
         <Switch>
-          <Route exact path="/" component={Home}/>
-          <Route path="/departments/:id" component={Department}/>
-          <Route path="/departments" component={Departments}/>
-          <Route path="/*" component={Error}/>
+          <Route exact path="/" component={Login} />
+          <Route path="/departments/:id" component={Department} />
+          <Route path="/departments" component={Departments} />
+          <Route path="/employee:id" component={Employee} />
```

Comment:
use lazy and suspense since you don't need to load the content until the user is signed in